config/locales/server.en.yml

```diff
@@ -13,6 +13,8 @@ en:
     disable_solved_education_message: "Disable education message for solved topics."
     accept_solutions_topic_author: "Allow the topic author to accept a solution."
     solved_add_schema_markup: "Add QAPage schema markup to HTML."
+    enable_solved_tags: "Allow users to select solutions if one of these tags are present"
```

Comment:
```suggestion
    enable_solved_tags: "Allow users to select solutions for the following tags"
```

I think we need to change the label for this too: 

<img width="676" alt="image" src="https://user-images.githubusercontent.com/368961/133677991-10b94b8b-df1d-4bbd-8eba-9b660ac228b1.png">

Maybe `Allow users to select solutions on all topics (when unchecked, solutions can be enabled per category or tag)`